website/docs/r/cdn_endpoint_custom_domain.html.markdown

```diff
@@ -75,7 +75,31 @@ The following arguments are supported:
 
 * `host_name` - (Required) The host name of the custom domain. Changing this forces a new CDN Endpoint Custom Domain to be created.
 
-~> **Note:** At this time only HTTP Custom Domains are supported.
+* `cdn_managed_https` (Optional) - A `cdn_managed_https` block as defined below. 
+
+* `user_managed_https` (Optional) - A `user_managed_https` block as defined below.
+
+~> **NOTE** Only one of `cdn_managed_https` and `user_managed_https` can be specified.
+
+!> **Warning** Due to the way the Azure API handles in-place updates on HTTPS settings for the Custom Domain of a CDN Endpoint, it is only possible to toggle these settings between disabled and enabled individually.
```

Comment:
this sounds like a provider issue that we should be handling through a multi-step update?